partcad/src/partcad/converters/cad_converter.py

```
@@ -0,0 +1,33 @@
+from .stl_converter import StlConverter
+from .step_converter import StepConverter
+from .. import logging
+# Import other converters as needed
+
+
+class CADConverter:
+    CONVERTERS = {
+        "step": StepConverter,
+        "stl": StlConverter,
+        # Add other formats as needed
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)